libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -159,7 +161,18 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
      * Id of elements (separated by space) for setting aria-labelledby for search input
      * Not shown in the UI, only visible by the screen-readers.
      */
-    @Input() ariaLabelledby: string; 
+    @Input() ariaLabelledby: string;
```

Comment:
```suggestion
    @Input()
    ariaLabelledby: string;
```
and in lines 169 and 175 too.

Reply:
fixed formatting for all.